src/content/en/tools/chrome-devtools/console/get-started.md

```diff
@@ -208,9 +208,6 @@ filter the **Console** to only show messages that you care about.
    filtering.
 
        {% framebox width="auto" height="auto" enable_widgets="true" %}
-
-         {% verbatim %}
```

Comment:
@kaycebasques - can you confirm that these should be removed? They are showing up on the [live page](https://developers.google.com/web/tools/chrome-devtools/console/get-started#filter), which I don't think is what you want.

Reply:
I think there was a reason I did this, but obviously it's not working as intended. So let's go for it...